otter/integration/lib/autoscale.py

```diff
@@ -132,6 +140,79 @@ def set_launch_config(self, rcs, launch_config):
             .addCallback(lambda _: rcs)
         )
 
+    def get_group_config(self, rcs):
+        """
+        Returns the group configuration recorded in the TestResources object.
+
+        :param TestResources rcs: The integration test resources instance.
+            This provides useful information to complete the request, like
+            which endpoint to use to make the API request.
+
+        :return:
+        """
+        # Retrieve the configuration from the stored response to group
+        # creation since that will include non-provided values
+        config = [g['group']['groupConfiguration'] for g in rcs.groups
+                  if g['group']['id'] == self.group_id]
+        # Since this should always provide a match for a valid scaling group,
+        # allow the exception to be raised if nothing is found.
+        if verbosity > 0:
+            print('ScalingGroup.get_group_config will return: ')
+            pp.pprint(config[0])
+        return config[0]
+
+    def update_group_config(self, rcs, name=None, cooldown=None,
+                            minEntities=None, maxEntities=None, metadata=None,
+                            replacement_config=None):
+        """
+        Update the group configuration of a scaling group. Ifa replacement
+        configuration is provide, it will be used to replace all values.
+        Otherwise, only the specified parameters will be altered.
+        """
+        # Get the old config
+        old_config = self.get_group_config(rcs)
+        new_config = {}
+        if replacement_config is not None:
+            new_config = replacement_config
+        else:
+            new_config["name"] = name or old_config["name"]
+            new_config["cooldown"] = cooldown or old_config["cooldown"]
+            new_config["minEntities"] = \
+                minEntities or old_config["minEntities"]
+            new_config["maxEntities"] = \
+                maxEntities or old_config["maxEntities"]
+            new_config["metadata"] = metadata or old_config["metadata"]
+
+        def record_results(_, new_config):
+            # Replace the stored value in slightly misnamed ScalingGroup
+            self.group_config["groupConfiguration"] = new_config
+            # Find the correct group from the list (by id)
+            for g in rcs.groups:
+                if g['group']['id'] == self.group_id:
+                    g['group']['groupConfiguration'] = new_config
+            if verbosity > 0:
+                print('Updated group_config with {}'.format(new_config))
+            return rcs
+
+        return (
+            treq.put(
+                "%s/groups/%s/config" % (
```

Comment:
Non-blocking issue: to be consistent with the rest of the code in the file, you should use `"".format()`` instead of the `%` operator.  No need to change it now, but if you modify this file again later on, you should change it to match.  Thanks!